.dev/sass/compat/woocommerce.scss

```diff
@@ -1,3 +1,7 @@
 .woocommerce-page ul.products li.product.primer-2-column-product {
 	width: 48.05%;
 }
+
+.woocommerce-page .primer-woocommerce .cart .qty {
+	padding: .18em;
```

Comment:
@EvanHerman I find that using a leading zero in these cases is more readable as the `.` can be overlooked at a glance: `0.18em`